config/settings.json

```diff
@@ -16,13 +16,15 @@
     "productsPerPage": 6,
     "menuTitle": "Menu",
     "menuLocation": "side",
-    "footerHtml": "&lt;h4 class=&quot;text-center&quot;&gt;Powered by expressCart&lt;/h4&gt;",
```

Comment:
Can you revert all changes in the `settings.json file` except the new additions?